add trd qc

bazinski opened the pull request:
Add trd to syn async and in to qc-workflows.sh to get the qc in global runs amongst other things.

bazinski commented:
ooops I forgot that, did it in a rush before the async qc meeting, fixing now.

bazinski commented:
Hi @chiarazampolli No the MC only has digits, I can fix that, but there was some python involved, it was a bit of an urgent fix at the time due to it failing for some or other reason. I could not follow and had more pressing things at the time to sort out.
I can come back to if you would like?

knopers8 commented on DATA/production/qc-async/trd.json:
```diff
@@ -0,0 +1,151 @@
+      "samplingConditions":[
+        {
+          "condition": "random",
+          "fraction": "0.9",
```
if you need almost all data (90%), you could consider attaching QC directly to the data, as illustrated in here:
https://github.com/AliceO2Group/QualityControl/blob/175baf49916e92552349654a8ce9b3fdb414cd42/Framework/basic-no-sampling.json#L34

knopers8 reviewed (commented):
Is there a reason not to add the `qc-async/trd.json` to the corresponding list in `qc-workflow.sh` as well?

bazinski commented:
Hi @knopers8 I had it on desktop, but not where i was doing the pr from. Fixed now.
qc tasks are direct in the json and I have removed the raw data task as opposed to just disabling it.
Also added it to the qc-workflows.sh
